packages/react/src/component-tests/IcCheckbox/IcCheckboxTestData.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { IcCheckbox, IcCheckboxGroup, IcTextField } from "../../components";
+
+export const Checkbox = () => {
+  return (
+    <div style={{ padding: "5rem" }}>
+      <IcCheckboxGroup
+        label="Select your extras"
+        name="default"
+        onIcChange={(ev) => console.log("onIcChange", ev.detail.value)}
+      >
+        <IcCheckbox
+          value="extra"
+          label="Extra shot (50p)"
+          onIcCheck={(ev) => console.log("onIcCheck", ev)}
+        />
+        <IcCheckbox value="Soya milk" label="Soya milk" checked />
+        <IcCheckbox value="keep cup" label="Takeaway cup" disabled />
+        <IcCheckbox value="other" label="Other">
+          <IcTextField slot="additional-field" label="Please let us know..." />
+        </IcCheckbox>
+        <IcCheckbox
+          additionalFieldDisplay="dynamic"
+          value="other"
+          label="Other"
+        >
+          <IcTextField slot="additional-field" label="Please let us know..." />
+        </IcCheckbox>
+      </IcCheckboxGroup>
+    </div>
+  );
+};
+
+export const CheckboxForm = () => {
+  const [submit, setSubmit] = useState("");
+  const handleSubmit = (event) => {
```

Comment:
Does `event` need a type? Getting this error / warning about it when running the Cypress tests:

<img src="https://github.com/mi6/ic-ui-kit/assets/111882109/0d427974-7f19-418c-8723-60bd77fbb5ed" width="500" >